src/helper/isImplementedCache.ts

```diff
@@ -20,7 +21,7 @@ export class IsImplementedCache {
     return undefined;
   }
 
-  add(eventName: string, isImplemented: boolean): void {
+  add(eventName: string, isImplemented: boolean, skillId?: string): void {
```

Comment:
Right now the code only adds the skillId as part of the entry (but does not use it at all afterwards). Although that should be relevant.

Reply:
added changes